Makefile

```diff
@@ -2,26 +2,22 @@ init:
 	go build -modfile=tools/go.mod -o bin/yq github.com/mikefarah/yq/v3
 	go build -modfile=tools/go.mod -o bin/statik github.com/rakyll/statik
 
-generate:
-	protoc \
-		-I api \
-		-I third_party/grpc-gateway/ \
-		-I third_party/googleapis \
-		--go_out=plugins=grpc,paths=source_relative:./versionpb \
-		--grpc-gateway_out=./versionpb \
-		--openapiv2_out=third_party/OpenAPI/ \
-		api/version.proto
+	curl -L https://github.com/uber/prototool/releases/download/v1.8.0/prototool-$(shell uname -s)-$(shell uname -m) -o ./bin/prototool
```

Comment:
1.10.0 is better, we just did not update in PMM yet

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/1/commits/25f036997b7fa30e441853baab641e00f843a120